ql/time/calendars/turkey.cpp

```diff
@@ -48,6 +48,8 @@ namespace QuantLib {
             || (d == 1 && m == May)
             // 19 may/ National Holiday
             || (d == 19 && m == May)
+			// 15 july / National Holiday
+			|| (d == 15 && m == July)
```

Comment:
If I read your issue correctly, this should be only since 2017, is that right?
```suggestion
            // 15 july / National Holiday (since 2017)
            || (d == 15 && m == July && y >= 2017)
```

Reply:
Yes it is true, thanks for the correction.